source/catalog.html

```diff
@@ -0,0 +1,188 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Седона | Фото и видео</title>
+  </head>
+  <body class="page__body">
+    <header class="main-header">
+      <a class="main-header__logo" href="index.html">
+        <img src="img/city-of-sedona-logo.svg" width="185" height="96" alt="Логотип города Седона.">
+      </a>
+
+      <nav class="main-header__nav">
```

Comment:
```suggestion
      <nav class="main-header__nav nav">
```
А лучше скопировать с index
header и footer от страницы к странице не отличаются